src/table/components/TableWrapper.jsx

```diff
@@ -93,8 +94,11 @@ export default function TableWrapper(props) {
   );
 
   useDidUpdateEffect(() => {
-    if (!keyboard.enabled) return;
-
+    // When nebula handles keyboard navigation,
+    // - when a user tabs to the chart and presses Enter or Space (the table is focused),
+    // the focus should be set to the last focused cell or the first cell in the head
+    // - when a user move the focus outside the table (the table is blurred),
+    // the last focused cell should be blurred (loses the focus)
```

Comment:
This is still inaccurate. All this does is to make sure to focus/blur when keyboard.active changes. This includes confirming selections from the selections toolbar, clicking to focus the table, cancelling selections, pressing escape in the body/header. I barely remember everything here, so we defensively need some kind of comments, but It is rather where it is called from that is complex, not what happens in here.